test/pummel/test-hash-seed.js

```diff
@@ -1,31 +1,25 @@
 'use strict';
 
 // Check that spawn child doesn't create duplicated entries
-require('../common');
-const Countdown = require('../common/countdown');
+const common = require('../common');
 const REPETITIONS = 2;
```

Comment:
Extremely minor nit: Can we s/REPETITIONS/kRepetitions here just for consistent styling of constants

Reply:
Sure, sounds good to me.